Automattic/wp-calypso
# Enabling FullStory in all environments.

This PR turns on the FullStory feature flag for all environments. The actual recording toggle switch lives in the FullStory admin interface and will remain off by default.

There's a companion diff that enables FullStory in wp-admin at D61423-code.

#### Testing instructions
* Checkout this branch and start Calypso.
* Navigate to the Calypso admin screen for one of your test sites.
* Open the dev console and turn on Calypso event debug messages (`localStorage.setItem( 'debug', 'calypso:*' );`).
* Filter the console output to show only FullStory events and confirm that you see FullStory debug messages as you navigate.
* Open the Network tab w/ the same FullStory filter and confirm you see the `fs.js` and `page` network activity.

Related to #52768
